src/coldfront_plugin_api/scim_v2/users.py

```diff
@@ -108,5 +109,8 @@ class UserDetail(APIView):
     permission_classes = [IsAdminUser]
 
     def get(self, request, username, format=None):
-        allocation = User.objects.get(username=username)
+        try:
+            allocation = User.objects.get(username=username)
+        except User.DoesNotExist:
+            raise Http404("Username not found")
```

Comment:
Does this return JSON or HTML?

Reply:
Entering an invalid username returns a HTML page that seems to be part of Django REST framework. It seems like it doesn't matter what I put as the input string to Http404, the REST framework is overriding whatever Django's Http404 page is.

Reply:
You don’t need to use Http404. There are other places in this file where I manually return a response with a specific status code.